custom-connectors/ReversingLabs Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ReversingLabs Intelligence",
+    "version": "1.0",
+    "description": "ReversingLabs continually processes goodware and malware files providing early intelligence about attacks before they infiltrate customer infrastructures. This visibility to threats “in-the-wild” enables preparation for new attacks and quickly identifies the threat levels of new files as they arrive. ReversingLabs enables more effective and efficient threat identification, development of better threat intelligence, and implementation of proactive threat hunting programs.",
+    "contact": {
+        "name": "ReversingLabs support",
+        "url": "https://support.reversinglabs.com/",
+        "email":"support@reversinglabs.com"
+    }
+  },
+  "host": "data.reversinglabs.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+	  {
```

Comment:
@vivanec Please fix all the indentations  in the swagger.

Reply:
Hi, done.